.github/workflows/build-tarball.yml

```diff
@@ -11,12 +11,11 @@ on:
       - v[0-9]+.x
 
 env:
-  FLAKY_TESTS: dontcare
+  PYTHON_VERSION: '3.10'
```

Comment:
Is this required? While Python 3 is to be recommended, Node.js 14 supports building with Python 2 and 3: https://github.com/nodejs/node/blob/v14.x/BUILDING.md#note-about-python-2-and-python-3

Reply:
I was already set to `3.10`, I simply moved it here to be consistent with the other workflow, see https://github.com/nodejs/node/pull/45126/files#diff-9e078b64804cbda5c87567835a5da2f1314788813ec5c5e9809a2f96519b9b01L18-L19.